lib/internal/test_runner/runner.js

```diff
@@ -216,6 +224,14 @@ class FileTest extends Test {
     if (item.data.details?.error) {
       item.data.details.error = deserializeError(item.data.details.error);
     }
+    if (item.type === 'test:bail') {
+      // <-- here we need to stop all the pending test files (aka subprocesses)
+      // To be replaced, just for poc
+      this.root.harness.testsProcesses.forEach((child) => {
+        child.kill();
+      });
+      return;
```

Comment:
replace with an abort signal?

Reply:
hey @atlowChemi, sure!

Reply:
We already have an existing signal passed to the spawned process, but that is a signal at the `Test` level IIRC. perhaps we can add a new controller on `root.harness`, and spawn with a `AbortSignal.any`

Reply:
Done! 
I moved the `AbortSignal.any` directly into `createTestTree` for consistency